policyengine_us/parameters/gov/bls/cpi/c_cpi_u.yaml

```diff
@@ -304,6 +304,7 @@ values:
   2031-02-01: 199.7 # 2032 value.
   2032-02-01: 203.7 # 2033 value.
   2033-02-01: 207.8 # 2034 value.
+  2034-02-01: 212.0 # 2035 Value (Indexed Using Values from Previous Year)
```

Comment:
indexed based on what?

Reply:
I took the previous two values and used the same inflation adjustment.

207.8/203.7 = 1.02013

207.8*1.02013 = 212.0 (when rounded)

Reply:
OK please note that in the comment, and apply sentence case